### ISSUE TITLE
incorrect forge download url

### ISSUE THREAD

**Issue by Spongman** [dizzyd/mcdex#10]
````
2017/08/28 02:47:30 failed to install Forge 10.13.4.1614: failed to read Forge installer: zip: not a valid zip file
````

the generated url used to download forge is sometimes wrong depending on the version.

sometimes the correct location looks like this:

`https://files.minecraftforge.net/maven/net/minecraftforge/forge/1.9-12.16.0.1886/forge-1.9-12.16.0.1886-universal.jar`

and sometimes like this:

`https://files.minecraftforge.net/maven/net/minecraftforge/forge/1.9-12.16.0.1885-1.9/forge-1.9-12.16.0.1885-1.9-installer.jar`

note the duplicated MC versions.

**Comment by dizzyd** [comment]
Sigh. What version of forge did you really want installed? I'll use that as a test case.

<rant>It's so obvious that some forge releases are built by hand. Not sure why TF they would do this for something so widely used....</rant>

**Comment by Spongman** [comment]
i don't remember exactly which one. project ozone 2 was the modpack from curse, i believe. i checked a bunch of download URLs, it looks like they only use those two schemes.

**Comment by dizzyd** [comment]
I've tried to reproduce this but haven't been able to. If you have a specific modpack URL that's failing, please re-open this issue with that so I can reproduce and fix. :)

**Comment by Spongman** [comment]
project ozone 2 was the one. i gave you all the info you need to fix.

just attempt to download from the URL scheme you currently do, if that fails, try the other one. done.

**Comment by dizzyd** [comment]
@Spongman I couldn't find a Project Ozone 2 on curseforge - do you have the download link?

**Comment by Spongman** [comment]
this is the one, i think: https://www.curseforge.com/minecraft/modpacks/project-ozone-2-reloaded

**Comment by dizzyd** [comment]
Fixed in v0.7.5. :) Thanks for the URL link.

**Comment by Spongman** [comment]
that still won't work for all forge versions.
